std/range/package.d

```diff
@@ -8647,3 +8647,237 @@ if (is(typeof(fun) == void) || isSomeFunction!fun)
 
     auto r = [1, 2, 3, 4].tee!func1.tee!func2;
 }
+
+template CommonElementType(Rs...)
```

Comment:
Is this package-qualified? If not, keep it `package` for now because it's not supposed to be public (not documented). Let's decide in a future pull whether this belongs to std.traits (or possibly std.range).